mlflow/pipelines/steps/automl/flaml.py

```diff
@@ -0,0 +1,124 @@
+import logging
+from typing import Dict, Any, Tuple
+
+import pandas as pd
+from sklearn.base import BaseEstimator
+
+import mlflow
+from mlflow import MlflowException
+from mlflow.models.evaluation.default_evaluator import _get_regressor_metrics
+from mlflow.pipelines.utils.metrics import _load_one_custom_metric_function, PipelineMetric
+
+_logger = logging.getLogger(__name__)
+
+AUTOML_DEFAULT_TIME_BUDGET = 10
+MLFLOW_TO_FLAML_METRICS = {
+    "mean_absolute_error": "mae",
+    "mean_squared_error": "mse",
+    "root_mean_squared_error": "rmse",
+    "r2_score": "r2",
+    "mean_absolute_percentage_error": "mape",
+}
+
+
+def get_estimator(
```

Comment:
Can we rename this to `get_estimator_and_best_params`?

Reply:
Done